Airports/EGSC/Ownership.txt

```diff
@@ -2,32 +2,35 @@ SECTOR:Only SCTWR 1:0:0
 OWNER:ESS:SCR:SCA:SCT
 
 SECTOR:Only SCTWR 2:0:0
-OWNER:ESS:SCR:SCA:TCNW:TCNE:LOREL:TCN:TCM:TCSW:TCSE:TCS:TCE:TC:LML:LME:LM:LC:LES:LEN:LE:LDL:LD:LSF:LS:LWN:LSC:L:LMEN:O
+OWNER:ESS:SCR:SCA:TCNE:TCNW:LOREL:TCN:TCM:TCSW:TCSE:TCS:TCE:TC:LML:LME:LM:LC:LES:LEN:LE:LDL:LD:LSF:LS:LWN:LSC:L:LMEN:O
 
 SECTOR:Only SCAPP:0:0
 OWNER:SCR:SCA
 
 SECTOR:SCTWR:0:2000
-OWNER:SCT:SCR:SCA:ESS:TCNW:TCN:TC:TCM:LML:LME:LM:LC:LSC:L
-ALTOWNER:Observing LTC_N_CTR:SCT:SCR:SCA:ESS:TCNW:O:LMEN:TCN:TC:LML:LME:LM:LC:LSC:L
-ALTOWNER:Observing LTC_CTR:SCT:SCR:SCA:ESS:TCNW:TCN:O:LMEN:TC:LML:LME:LM:LC:LSC:L
-ALTOWNER:Observing LON_M_CTR:SCT:SCR:SCA:ESS:TCNW:TCN:TC:LML:LME:O:LMEN:LM:LC:LSC:L
-ALTOWNER:Observing LON_C_CTR:SCT:SCR:SCA:ESS:TCNW:TCN:TC:LML:LME:LM:O:LMEN:LC:LSC:L
-ALTOWNER:Observing LON_SC_CTR:SCT:SCR:SCA:ESS:TCNW:TCN:TC:LML:LME:LM:LC:O:LMEN:LSC:L
-ALTOWNER:Observing London FIR:O:LMEN:GWT:GWR:ESS:TCNW:TCN:TC:LML:LME:LM:LC:LSC:L
-ALTOWNER:Mentoring TWR/APP:SCTM:SCT:SCRM:SCAM:GWR:ESSM:ESS:TCNW:TCN:TC:LML:LME:LM:LC:LSC:L
+OWNER:SCT:SCR:SCA:SSR:ESS:TCNE:TCN:TC:SABER:TCE:TCE:LEN:LE:LC:LSC:L
```

Comment:
Also TCE is in here twice, maybe unintentionally?

Reply:
@danielbutton because ESS is still a position that covers EGSS, EGGW and EGSC top-down - will be explained in procedure change post, but is defined here https://github.com/VATSIM-UK/UK-Sector-File/blob/00a58cd643c89787c962823242603ebdad1aca7b/Airports/EGSS/Positions.txt#L7